pages/common/pueue.md

```diff
@@ -3,34 +3,18 @@
 > Pueue is a command-line task management tool for sequential and parallel execution of long-running tasks.
 > More information: <https://github.com/Nukesor/pueue>.
 
-- Add a command to the task list:
+- Show general help and available subcommands:
 
-`pueue add -- "{{command}}"`
+`pueue --help`
 
-- List tasks in the task list:
+- Check pueue version:
 
-`pueue status`
+`pueue --version`
 
-- Send data to a task's stdin:
+- Execute a pueue subcommand (like `add`, `parallel`, etc.):
 
-`pueue send {{task_id}} {{"hello"}}`
+`pueue {{subcommand}}`
 
-- View a task's stdout and stderr, as well as basic information about that task:
+- Show more example on a pueue subcommand:
 
-`pueue log {{task_id}}`
-
-- Create a task group:
-
-`pueue group --add {{group_name}}`
-
-- Kill a task:
-
-`pueue kill {{task_id}}`
-
-- Set maximum amount of parallel tasks (queued tasks are started as needed to meet this limit):
-
-`pueue parallel {{number_of_parallel_tasks}}`
-
-- Edit the command line of a stopped task in the default editor (as specified by `$EDITOR`):
-
-`pueue edit {{task_id}}`
+`tldr pueue-{{subcommand}}`
```

Comment:
I'm not sure if we should include this example as it doesn't demonstrate `pueue` functionality, but at least we can remove the dash as it's not necessary:
```suggestion
`tldr pueue {{subcommand}}`
```